apps/acrs/index.js

```diff
@@ -180,7 +205,24 @@ module.exports = {
       locals: { showSaveAndExit: true }
     },
     '/parent-summary': {
-      fields: [],
+      behaviours: [AggregateSaveUpdate, ParentSummary, LimitParents, SaveFormSession],
+      aggregateTo: 'referred-parents',
+      aggregateFrom: [
+        'parent-full-name',
+        'parent-phone-number',
+        'parent-email',
+        'parent-date-of-birth',
+        'parent-country',
+        'parent-evacuated-without-reason'
+      ],
+      titleField: 'parent-full-name',
+      addStep: 'parent-details',
+      addAnotherLinkText: 'parent',
+      locals: { showSaveAndExit: true },
+      continueOnEdit: false,
+      template: 'parent-summary',
+      backLink: 'parent',
+      aggregateLimit: 2,
```

Comment:
recommend replacing magic number with `const PARENT_LIMIT = 2;`

Reply:
It's a decent shout - we can store all the limit values in one section making it easier to find for updates if they happen

Reply:
Updated in latest commit